Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -926,6 +971,8 @@ static void logProcessPoolState(const WebProcessPool& pool)
 #if PLATFORM(IOS_FAMILY)
     [[NSNotificationCenter defaultCenter] removeObserver:m_accessibilityEnabledObserver.get()];
     [[NSNotificationCenter defaultCenter] removeObserver:m_applicationLaunchObserver.get()];
+    auto notificationName = adoptNS([[NSString alloc] initWithCString:kGSEventHardwareKeyboardAvailabilityChangedNotification encoding:NSUTF8StringEncoding]);
+    removeCFNotificationObserver((__bridge CFStringRef)notificationName.get());
```

Comment:
How does this work when there is more than 1 process pool?

Reply:
This call will only remove the observer for the calling process pool. Other process pools will still be observing this notification until they call this function. This is because each process pool is calling `CFNotificationCenterAddObserver` with a unique observer parameter.

Reply:
Oh, there is logic in `removeCFNotificationObserver()`.. We weren't passing a pointer to `this` so I was confused.